src/pom.xml

```diff
@@ -1438,7 +1427,6 @@
       <exclude>${test.exclude.pattern}</exclude>
      </excludes>
      <forkMode>once</forkMode>
-     <argLine>-Xmx${test.maxHeapSize} -enableassertions ${jvm.opts} -Djava.awt.headless=${java.awt.headless} -Dsun.java2d.d3d=${sun.java2d.d3d} -DremoteOwsTests=${remoteOwsTests} -DquietTests=${quietTests} -Dorg.geotools.image.test.enabled=${image.tests} -Dorg.geotools.image.test.interactive=${interactive.image} -Duser.timezone=${user.timezone} -Dwindows.leniency=${windows.leniency} -XX:+TieredCompilation -XX:TieredStopAtLevel=1</argLine>
```

Comment:
This is certainly bad?

Reply:
That got moved to a property, so that both jacoco and surefire can use it without clobbering over each other (the jacoco setup needs to append some stuff to argline during setup, but was getting overwritten by the surefire plugin config).
Since that property is only used by those plugins, this should not have an effect when they are not being used.
As far as I can tell, the surefire tests are still correctly picking up the argline (I would expect there to be at least one test failure if this were not the case)